docs/how-to-guides/develop-a-tool/use-file-path-in-python-tool-and-custom-tool.md

```diff
@@ -0,0 +1,54 @@
+# Use FilePath as input in python tool and custom tool
+
+In this document, we will guide you through the detailed process of how to use FilePath as input in python tool or custom tool.
+
+## Prerequisites
+
+Since FilePath is still in test currently, you need to install private version SDK and Extension.
+
+1. Download private version promptflow SDK
+    ```
+    pip install "promptflow==0.0.106113823" --extra-index-url https://azuremlsdktestpypi.azureedge.net/test-promptflow/
+    ```
+
+2. Download this [VSCode Extension](https://aka.ms/pfvsctest), install from VSIX: 
+    ![install from vsix](../../media/how-to-guides/develop-a-tool/install_from_vsix.png)
+
+## Use FilePath in python tool
```

Comment:
The gen meta difference between package tool (custom tool you mentioned) and python tool is not what we focus in this doc. From tool interface perspetive, they are the same. Our focus of this doc is how to write a tool with file path type.

I think you could add a tool with file path input in hello-world/tools rather than modify existing one.

Reply:
updated